lib/rcode5/component.rb

```diff
@@ -0,0 +1,10 @@
+module Rcode5
+  class Component
+    include ActiveModel::Model
+    include ActiveModel::Conversion
+
+    def to_partial_path
+      "/" + self.class.name.underscore
```

Comment:
not sure how this works if this template is the same name as a rails template.  can we count on long component names for uniqueness?